### PR TITLE
Explosion: Push by default and stronger push

### PR DESCRIPTION
**Pull request by weluvgoatz**
This pull request makes the push explosion effect stronger, and also applies it to bombs, haywires and goldbombs so their explosions push back players.  Please test, thanks!

### CODE REVIEW

File: src/object/explosion.cpp [hunk 1]
```diff
@@ -90,9 +90,7 @@ Explosion::explode()
 
       /* The force decreases with the distance squared. In the distance of one
        * tile (32 pixels) you will have a speed increase of 150 pixels/s. */
-      float force = 150.0f * 32.0f * 32.0f / (distance * distance);
-      if (force > 200.0f)
-        force = 200.0;
+      float force = 150.0f * 64.0f * 64.0f / (distance * distance);
```
[the remark this answers is not in the document]
Comment (weluvgoatz) [reply src/object/explosion.cpp]: fixed because no cap could cause unexpected behaviour

File: src/object/explosion.cpp [hunk 2]
```diff
@@ -76,7 +76,7 @@ Explosion::explode()
 
   if (push) {
     Vector center = m_col.m_bbox.get_middle ();
-    auto near_objects = Sector::get().get_nearby_objects (center, 10.0 * 32.0);
+    auto near_objects = Sector::get().get_nearby_objects (center, 128.0 * 128.0);
```
Comment (HybridDog) [inline src/object/explosion.cpp]: I think the 32.0 factor is there to have the value in tiles, so 10.0 * 32.0 means 10.0 tiles.
Here the argument corresponds to 512 tiles: 128.0*128.0=512.0*32.0

**Comment by HybridDog** [comment]
After this change it may be possible that Tux can push himself to places where he should not be, so in some levels he may reach the goal earlier than he should. I don't know if it is important to avoid possible compatibility problems in existing levels.

**Comment by weluvgoatz** [comment]
The bombs cannot cause significant pushes to make players move that far upwards, save for shortfuses.  Shortfuses are hardly used anyway, and the levels where they are used use them sparingly so they are not used in this way (Owls' Skydive Commando, some Bonus 3/4 levels)

File: src/object/explosion.cpp (hunk 2 above)
Comment (weluvgoatz) [reply src/object/explosion.cpp]: I might change/fix that, but bigger number = more significant explosion :P

**Comment by HybridDog** [comment]
> The bombs cannot cause significant pushes to make players move that far upwards […]

I've tested it (your explosion branch); the bomb can push up Tux far into the sky: 
https://streamable.com/1f7r2

**Pull request merged**